styles.css

```diff
@@ -560,17 +531,192 @@ textarea {
   flex-direction: row;
   align-items: center;
   padding: 12px 16px;
-  position: absolute;
   width: 140px;
   height: 48px;
   background: #fff;
   border-radius: 8px;
-  font-family: "Poppins", Verdana, Tahoma, sans-serif;
   font-style: normal;
   font-weight: 500;
   font-size: 17px;
   line-height: 24px;
-  text-align: center;
   letter-spacing: 0.03em;
   color: #6070ff;
 }
+
+@media screen and (min-width: 768px) {
+  .main-section {
+    width: cover;
+    height: 920px;
+    background-image: url(./images/Header\ bg.svg);
+    background-size: 95%;
+    background-position: center 40%;
+    padding: 25% 18% 25% 18%;
+  }
+
+  .logo {
+    margin: 25px auto 0 155px;
+  }
+
+  .menu {
+    margin: 25px 155px 0 0;
+  }
+
+  .menu div {
+    display: none;
+  }
+
+  .menu ul {
+    display: flex;
+    gap: 20px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 13px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  h1 {
+    font-size: 48px;
+    line-height: 60px;
+    margin-bottom: 1rem;
+  }
+
+  .main-section p {
+    font-size: 17px;
+    line-height: 28px;
+    margin-bottom: 1.5rem;
+  }
+
+  .lc {
+    font-size: 13px;
+  }
+
+  .icons {
+    margin-top: -0.6rem;
+  }
+
+  .works {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .projects {
+    flex-direction: row;
+    height: 22%;
+    width: 80%;
+    justify-content: space-between;
+    padding-top: 30px;
+  }
+
+  .img {
+    width: 45%;
+    height: 100%;
+    margin-right: 2rem;
+  }
+
+  .projectInfo {
+    width: 55%;
+    margin-bottom: 5%;
+  }
+
+  .aboutProject {
+    width: 90%;
+    height: 100%;
+  }
+
+  .reverse {
+    flex-direction: row-reverse;
+  }
+
+  .pointer {
+    background-image: url(./images/hand-cursor\ 1.png);
+    background-repeat: no-repeat;
+    background-position: 87% 100%;
+  }
+
+  .ruby {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 12px;
+    width: auto;
+    height: 24px;
+    background: #ebebff;
+    border-radius: 8px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+  }
+
+  .aboutme {
+    display: grid;
+    grid-template-rows: 3fr 3fr;
```

Comment:
- [ ] Please note that in the design, the left section is 40% of the width and the right side is 60% of the width. You have made them both 50%. Please change this to match the design.

_**figma**_
![figma-1](https://user-images.githubusercontent.com/99656687/213405838-bc8180f9-f193-4ddc-bbe9-95c984daab81.png)

_**Your desgn**_
![figma-2](https://user-images.githubusercontent.com/99656687/213406048-b3274a27-0ede-4e2a-852f-f5ed264bc636.png)